src/components/List/List.tsx

```diff
@@ -76,11 +80,25 @@ class List extends UIComponent<Extendable<IListProps>, any> {
   // List props that are passed to each individual Item props
   static itemProps = ['debug', 'selection', 'truncateContent', 'truncateHeader', 'variables']
 
+  private focusContainer = ContainerFocusHandler.create(this)
+
+  actionHandlers: AccessibilityActionHandlers = {
+    moveNext: this.focusContainer.moveNext.bind(this.focusContainer),
```

Comment:
nit: just as a matter of subsequent refactoring, we could provide a method that will fetch all four from focus container as an object, so that the code will look something like

```js
actionHandlers: AccessibilityActionHandlers = {
  ...this.focusHandler.getNavigationHandlers()
}
```

Reply:
let's do it as the next stage of refactoring, as this is more complex than it looks. MoveFrist, MoveLast might be not used in some of the scenarios, some other scenarios might require PageUp and PageDown in addition to what is already defined.